### ISSUE TITLE
Support qmd files

### ISSUE THREAD

**Issue by zareami10** [cursorless-dev/cursorless#880]
Currently commands don't work in `.qmd` files (see https://quarto.org/), but since it's more or less markdown it would be nice if we could have basic supports for it (e.g. without the additional features).

**Comment by pokey** [comment]
Yeah shouldn't be too hard.  Can basically just see how we handle eg jsonc today: we just reuse the json parser, and it mostly just works.  Can just grep in https://github.com/cursorless-dev/vscode-parse-tree and https://github.com/cursorless-dev/cursorless for `jsonc` to see how it works; should be easy fix

**Comment by zareami10** [comment]
> Yeah shouldn't be too hard. Can basically just see how we handle eg jsonc today: we just reuse the json parser, and it mostly just works. Can just grep in https://github.com/cursorless-dev/vscode-parse-tree and https://github.com/cursorless-dev/cursorless for `jsonc` to see how it works; should be easy fix

One thing I have noticed is that no Cursorless commands work when [Quarto's extension](https://marketplace.visualstudio.com/items?itemName=quarto.quarto) is active. Not really sure who's the culprit and/or how to go about debugging that.

**Comment by BlueDrink9** [comment]
> > Yeah shouldn't be too hard. Can basically just see how we handle eg jsonc today: we just reuse the json parser, and it mostly just works. Can just grep in https://github.com/cursorless-dev/vscode-parse-tree and https://github.com/cursorless-dev/cursorless for `jsonc` to see how it works; should be easy fix
> 
> One thing I have noticed is that no Cursorless commands work when [Quarto's extension](https://marketplace.visualstudio.com/items?itemName=quarto.quarto) is active. Not really sure who's the culprit and/or how to go about debugging that.

I was experiencing the issue that any cursorless (or indeed, vscode-talon) spoken command would run a bunch of notebook r code instead of the command. It turns out this is because of a keybinding clash between quarto and the command server. Remap the quarto command, or implement by PRs
